proposals/proposal-0001.md

```diff
@@ -0,0 +1,223 @@
+   submissions without sufficient motivation may be rejected outright.
+1. Rationale - The rationale fleshes out the specification by describing what
+   motivated the design and why particular design decisions were made. It should
+   describe alternate designs that were considered and related work, e.g. how
+   the feature is supported in other projects.
+1. Backwards Compatibility - All proposals that introduce backwards
+   incompatibilities must include a section describing these incompatibilities
+   and their severity. The proposal must explain how the author proposes to deal
+   with these incompatibilities. proposal submissions without a sufficient
+   backwards compatibility treatise may be rejected outright.
+1. Reference Implementation - The reference implementation must be completed
+   before any proposal is given status "Final", but it need not be completed
+   before the proposal is accepted. While there is merit to the approach of
+   reaching consensus on the specification and rationale before writing code,
+   the principle of "rough consensus and running code" is still useful when it
+   comes to resolving many discussions of API details.
+
+The final implementation must include test code and documentation appropriate
+for the Helm project's reference or the community at large.
+
+## Proposal header preamble
+
+Each proposal must begin with a [YAML][yaml] style header preamble. The headers
+must appear in the following order:
+
+Headers marked with "*" are optional and are described below. All other headers
+are required.
+
+```yaml
+---
+proposal: <proposal number>
+title: "<proposal title>"
+authors: [ "<list of authors' real names and optionally, email addresses>" ]
+created: 2020-08-13
+* helm-version: "<version number>"
+* requires: [ <proposal numbers> ]
+* replaces: <proposal number>
+* superseded-by: <proposal number>
+---
+```
+
+The `authors` header lists the names, and optionally the email addresses of all
+the authors/owners of the proposal. The format of the author header value must
+be
+
+```yaml
+authors: [ "John Doe <john@example.com>", "Jane Doe <jane@example.com>" ]
+```
+
+If the email address is included, and
+
+```yaml
+authors: [ "John Doe", "Jane Doe" ]
+```
+
+If the address is not given.
+
+The `created` header records the date that the proposal was published to the
+Helm project on Github. The header should be in [RFC 3339][rfc3339] format, e.g.
+2020-08-13.
+
+Proposals will typically have a `helm-version` header which indicates the
+version of Helm that the feature will be released with. proposals without a
+`helm-version` header indicate interoperability standards that will initially be
+supported through external libraries and tools, and then supplemented by a later
+proposal to add support to the project's standard library or Command Line
+Interface. Informational and process proposals do not need a `helm-version`
+header.
```

Comment:
Is this one of those things can is added later, after the implementation has shipped? If so, it might be worth spelling that out.